CSharpMath.CoreTests/TypesetterTests.cs

```diff
@@ -68,15 +68,15 @@ public class TypesetterTests {
         });
     [Theory]
     [InlineData("%\n1234", "1234")]
-    [InlineData("12.b% comment ", "12.b")]
+    [InlineData("12.b% comment ", "12.𝑏")]
     [InlineData("|`% \\notacommand \u2028@/", "|`@/")]
-    public void TestComments(string latex, string resultText) =>
+    public void TestComments(string latex, string text) =>
```

Comment:
Is it known what this test is for? If not, we should delete. If so, we should rename/document.

Reply:
Makes sure that comments are ignored correctly when generating displays

Reply:
OK so rename to CommentsAreIgnoredWhenGeneratingDisplays or add xml comment `Tests that comments are ignored correctly when generating displays

Reply:
I think just inserting the word "Ignore" will be sufficient here.